vulnerability/internal/entities/management/filter.go

```diff
@@ -92,10 +99,18 @@ func (f *Filter) setRepositoryIDFromRequest(r *http.Request) error {
 	return nil
 }
 
-func (f *Filter) setVulnerabilityFilters(r *http.Request) {
+func (f *Filter) setVulnerabilityFilters(r *http.Request, validateVulnFile bool) error {
```

Comment:
This validations should be in the `Validate` func

Reply:
done